### ISSUE TITLE
Also support "main" as default branch for repo normalization

### ISSUE THREAD

**Issue by TimidRobot** [creativecommons/ccos-scripts#48]
## Describe the bug

> Master-slave is an oppressive metaphor that will and should never become fully detached from history. Aside from being unprofessional and oppressive it stifles participation

([1.1. Master-slave - Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html#rfc.section.1.1))

## Expected behavior

The [`normalize_repos/normalize_repos.py`](https://github.com/creativecommons/ccos-scripts/blob/master/normalize_repos/normalize_repos.py) script should be updated to support both `main` and `master` as the primary branch.

## Additional context
- [Easily rename your Git default branch from master to main - Scott Hanselman](https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx)
- [Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html)

**Comment by kgodey** [comment]
Marking as blocked until we discuss this in the engineering meeting.

**Comment by TimidRobot** [comment]
## Implementation Details
1. This issue
2. Project maintainers will decide per repository whether they want to move to using `main`. Once this issue has been resolved, @TimidRobot will open a new issue to track those repositories.
3. Engineering team will re-evaluate in ~6 months based on the experience of using `main` in those repositories

**Comment by kgodey** [comment]
We're not taking any action on this until we have different default branches.

**Comment by TimidRobot** [comment]
There are two considerations:
1. Renaming: repositories that were created with `master` as the *default branch* name. We are waiting on GitHub to improve their tooling prior to renaming branches.
2. Creating: new repositories. This issue prevents new repositories from being created with `main` as the *default branch* name.

**Comment by akmadian** [comment]
I'd like to take this on

**Comment by dhruvkb** [comment]
Hey @akmadian, please do. We already have two repos (that I know of) that have either `main` (`creativecommons/og-image-generator`) or `develop` (`cc-licenses-data`) as the primary branch instead of `master`. We should update the code to get the primary branch from the repo itself.

**Comment by akmadian** [comment]
Sounds good! I'll start within the next couple days : )

**Comment by dhruvkb** [comment]
Just in case you need some help, here is a [the relevant documentation](https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.default_branch).
